tailwind.config.js

```diff
@@ -13,5 +15,11 @@ export default {
         './resources/views/filament/**/*.blade.php',
         './vendor/filament/**/*.blade.php',
         './vendor/awcodes/filament-tiptap-editor/resources/**/*.blade.php',
+        "./node_modules/flowbite/**/*.js",
+    ],
+    plugins: [
+        forms,
+        typography,
+        // require('flowbite/plugin')
```

Comment:
Does this need to stay?

Reply:
For now, yes. I added a TODO to fix it later